admin_guide/sysctls.adoc

```diff
@@ -118,6 +118,11 @@ situations such as high-performance or real-time application tuning.
 If you want to use unsafe sysctls, cluster administrators must enable them
 individually on nodes. They can enable only namespaced sysctls.
 
+There are two methods to control which sysctl options can be defined:
+
+- `forbiddenSysctls` option excludes specific sysctls.
+- `allowedUnsafeSysctls` option controls specific needs such as high performance or real-time application tuning.
```

Comment:
There is IBM Style Guide (ISG) guidance to not start a sentence with a lowercase word when possible. I suggest adding "The" at the beginning of each option.

Reply:
Also, I noticed that "high performance" is hyphenated previously in line 116. I prefer your version, as I think "high performance" is a noun and not an adjective describing app tuning (i.e., "high-performance application tuning"). So maybe just drop the hyphen in the previous instance?

Reply:
Great catches. I applied and pushed those changes. I was a little hesitant on the hyphen, but reading in context, it definitely is a noun.

@sunilcio can you review these changes and let me know if I covered everything accurately? I need to make another ticket, I believe, to update 4.5+ with these changes if they are validated. Thanks!